nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_umac_if.h

```diff
@@ -13,6 +13,7 @@
 #ifndef __HOST_RPU_UMAC_IF_H
 #define __HOST_RPU_UMAC_IF_H
 
+#include <stdbool.h>
```

Comment:
There is a task to remove all standard headers from OSAL code, this causes issues with Linux. Can you please also raise a Linux driver PR to make sure it works?

Reply:
Removed the stdbool header inclusion since variable type has been changed to unsigned char.